domains/authorization/Authorization.API/API/ValueObjects/Username.cs

```diff
@@ -2,15 +2,20 @@
 
 namespace API.ValueObjects;
 
-public class Name : ValueObject
+public class Username : ValueObject
 {
     public string Value { get; }
 
-    public Name(string value)
+    private Username(string value)
```

Comment:
Nit: UserName rather than Username. It is a users name, and not a username for logging into a system

Reply:
changed